runtime/doc/api.txt

```diff
@@ -1602,19 +1602,20 @@ nvim_buf_add_highlight({buffer}, {src_id}, {hl_group}, {line},
                 marks do.
 
                 Namespaces are used for batch deletion/updating of a set of
-                highlights. To create a namespace, use |nvim_create_namespace|
-                which returns a namespace id. Pass it in to this function as
-                `ns_id` to add highlights to the namespace. All highlights in
-                the same namespace can then be cleared with single call to
-                |nvim_buf_clear_namespace|. If the highlight never will be
-                deleted by an API call, pass `ns_id = -1` .
+                highlights. To create a namespace, use
```

Comment:
The text here and below is automatically generated from the source, so these edits should be fixed there.

Reply:
OK, I corrected the appropriate source file and run `scripts/gen_vimdoc.py`. The result documentation text file is the same.